tests/chaos/probes/nats_probe.py

```
@@ -1,3 +1,44 @@
+
+            nats_url = os.getenv("NATS_URL", "nats://localhost:4222")
+            # Connect with a shorter timeout for chaos testing probes
+            nc = await nats.connect(
+                nats_url, connect_timeout=5, max_reconnect_attempts=1
+            )
+            js = nc.jetstream()
+
+            # This throws if the stream does not exist or NATS is unavailable
+            await js.stream_info(stream_name)
+            result["healthy"] = True
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)